web/sites/default/config/default/views.view.affinity_groups_with_tag.yml

```diff
@@ -388,7 +388,7 @@ display:
           element_wrapper_type: ''
           element_wrapper_class: ''
           element_default_classes: true
-          empty: ''
+          empty: '<a href="/user/login?destination=/affinity-groups{{ view_node }}" class="btn btn-sm btn-primary"><small>Login to join</small></a>'
```

Comment:
Not sure if this is written correctly. Should the destination still have /affinity-groups in it?

Reply:
I think not, but can you recreate this branch as md-1565 so an md- branch will be created? Also when you create the pull request with an md- branch please edit the PR template so the link to the md- branch works.

Reply:
Yes I can do that I had a previous branch called md-1565 so it would not let me name it that. Also is that for this area? ( [http://md-{{](http://md-%7B%7B/) ticket number }}-accessmatch.pantheonsite.io )

Reply:
I tried putting in the link but it shows 404 so I do not think its correct

Reply:
The branch can be anything that starts with md- with no other special characters and less than 11 characters total. You can also remove the branch other branch if it's not being used any longer. That is the part of the template that you should replace with the branch name so that it is easy to link to the md- environment.

Reply:
I deleted the previous branch and was able to rename this branch to md-1565 but I am not sure how to update it to refer to that on github

Reply:
You can close this PR and create a new one after pushing the md-1565 branch.